admin/requirements.txt

```diff
@@ -1 +1 @@
--e git://github.com/camptocamp/c2cgeoform.git#egg=c2cgeoform
+-e git://github.com/camptocamp/c2cgeoform.git@731d8af#egg=c2cgeoform
```

Comment:
To monitor...

Reply:
Yes this should be updated when https://github.com/camptocamp/c2cgeoform/pull/113 is merged.